go/signedexchange/signedexchange.go

```diff
@@ -192,6 +192,9 @@ func (e *Exchange) decodeResponseMap(dec *cbor.Decoder) error {
 		if err != nil {
 			return err
 		}
+		if string(key) != strings.ToLower(string(key)) {
```

Comment:
Please assign `string(key)` to a temporary variable and use it here and line 209 ([]byte -> string conversion makes a copy).

Reply:
OK, I'll assign it into `key_str`.